src/Tests/SettingsTest.php

```diff
@@ -29,7 +29,7 @@ protected function setUp() {
 
     $permissions = [
       'access administration pages',
-      'administer acquia lift',
+      'administer acquia_lift',
```

Comment:
Not sure if this is a change we can still make, but the general sense I have is that modules keep their full name if we're talking about global permissions.